### PR TITLE
turn on popolo export for SYE in cron

### PR DESCRIPTION
**Pull request by struan**

### CODE REVIEW

File: conf/crontab.ugly [hunk 1]
```diff
@@ -60,6 +60,9 @@ MAILTO=cron-!!(*= $site *)!!@mysociety.org
 !!(* } elsif ($vhost eq 'nigeria.mzalendo.mysociety.org') { *)!!
```
Comment (mhl) [inline conf/crontab.ugly]: I think this is an old vhost which has now been removed, the condition should now be based on the vhost `www.shineyoureye.org` - it looks like this means that the feedback reports haven't been happening for ShineYourEye (oops :frowning_face:)

Comment (struan) [inline conf/crontab.ugly]: Fixed now.

File: conf/crontab.ugly [hunk 2]
```diff
@@ -57,7 +57,7 @@ MAILTO=cron-!!(*= $site *)!!@mysociety.org
 # Check for any recurring data problems:
 47 4 * * * !!(*= $user *)!! run_management_command kenya_check_for_recurring_problems
 
-!!(* } elsif ($vhost eq 'nigeria.mzalendo.mysociety.org') { *)!!
+!!(* } elsif ($vhost eq 'www.shineyoureye.com') { *)!!
```
Comment (mhl) [inline conf/crontab.ugly]: It's `.org` :)

Comment (struan) [inline conf/crontab.ugly]: Which is amazing as I got it correct two lines later :(

Fixed now. I will squash all this embarrassment down before I merge :|

**Comment by mhl** [comment]
:+1: (assuming you're going to squash these ;))

**Pull request merged**